src/logic/Operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from "big.js";
+
+export default function Operate(numberOne, numberTwo, operation) {
+    const one = Big(numberOne);
+    const two = Big(numberTwo);
+    if (operation == "/"){
+        return one.div(two).toString();
+    }
+    if (operation == "*"){
+        return one.times(two).toString();
+    }
+    if (operation == "-"){
+        return one.minus(two).toString();
+    }
+    if (operation == "+"){
+        return one.plus(two).toString();
+    }
+    if (operation == "%"){
+        return one.mod(two).toString();
+    }
```

Comment:
- [ ] Always use strict equality `===` for your comparisons. Kindly modify all conditions